README.md

```diff
@@ -1,3 +1,45 @@
-# resin.io docs
+Resin Documentation
+---------
```

Comment:
The dashes should extend to the same width as the title. Also, it's a good practice to use equal signs instead of dashes for the main title:

```
Resin Documentation
===================
```